Doc/Makefile

```diff
@@ -107,7 +107,7 @@ clean:
 
 venv:
 	$(PYTHON) -m venv venv
-	./venv/bin/python3 -m pip install -U Sphinx
+	./venv/bin/python3 -m pip install sphinx~=1.5.6
```

Comment:
I think, we depend upon the latest version intentionally.

So, after this lands, #1613 can be merged and we can revert back to using the latest version?